src/main/java/org/traccar/api/resource/SessionResource.java

```diff
@@ -81,6 +81,7 @@ public User get(@QueryParam("token") String token) throws StorageException, IOEx
             LoginResult loginResult = loginService.login(token);
             if (loginResult != null) {
                 User user = loginResult.getUser();
+                request.getSession().invalidate();
```

Comment:
1. Have you checked that this works correctly?
2. Are you sure you've added it to all the places?
3. If there are more places, it might make sense to add some helper method for this.

Reply:
No, let's do the refactoring here. It's a pretty small change.

Reply:
I'd like to know more about your request. In which project folder would you place the extension method and the class? Which lines do you want to include in the method?

Reply:
Thanks for the feedback. I've added a helper method for this duplicate code.